monitoring/snippets/v3/uptime-check-client/snippets.py

```diff
@@ -129,7 +179,12 @@ def get_uptime_check_config(config_name):
 # [START monitoring_uptime_check_delete]
 # `config_name` is the `name` field of an UptimeCheckConfig.
 # See https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.uptimeCheckConfigs#UptimeCheckConfig.
-def delete_uptime_check_config(config_name):
+def delete_uptime_check_config(config_name: str) -> None:
```

Comment:
Return a value to test, instead of capturing output. Boolean result seems reasonable.

Reply:
This method is not parsed. Changing its return type will result in refactoring the whole snippet client.

Reply:
This methods calls API method that does not return value.

Reply:
It would make sense to update the function to return `true` if operation succeeded and `false` otherwise. Nevertheless this might be out of scope for the fixit, so if it's something you'd like to postpone please log a bug for a future improvement in this sample.

Reply:
What would be the recommended way to handle unsuccessful operation of the underlying client library does not provide the status? Showing a simple try/catch all does not consider a best practice AFAIK. Would you like to provide a reference sample?

Reply:
A failure raises an exception, I expect. An assertion that calling it does not raise an exception would be the way to test it, unless the function itself would rather handle the exception and return a boolean. Since there was no test for not raising an exception before, it's fine as it is for this PR.